src/carousel/carousel.ts

```diff
@@ -180,6 +207,15 @@ export class NgbCarousel implements AfterContentChecked,
   ngAfterContentInit() {
     // setInterval() doesn't play well with SSR and protractor,
     // so we should run it in the browser and outside Angular
+
+    // Initialize the 'active' class (not managed by the template)
+    this._ngZone.onStable.pipe(take(1)).subscribe(() => {
```

Comment:
it seems strange that we need the `onStable` here. Not necessary for the navs, for example

Reply:
The elements (the slides) are not here yet, but I think I can use the `AfterViewInit` hook instead.